src/components/EditSnack/EditSnack.js

```diff
@@ -0,0 +1,36 @@
+import {connect} from 'react-redux';
+import React from 'react';
+import {setActualItem} from '../ConfirmationBox/actions';
+import ButtonList from '../ButtonList/ButtonList';
+
+class EditSnack extends React.Component {
+	render() {
+		return (
+			<ButtonList
+				items={Object.values(this.props.storeList)}
```

Comment:
We could do this mapping in `mapStateToProps` so that the component didn't have to care about it,

e.g.

`mapStateToProps = { items: Object.values(state.storeList) }`